src/backend_tools/db_provider.ts

```diff
@@ -79,11 +82,12 @@ class DbProvider {
   }
 
   async getClubsByName(name: string): Promise<Club[]> {
-    const clubRef = collection(this.db, this.clubPath);
-    const q: Query<DocumentData> = query(clubRef, where('name', '==', name));
-    const snapshot = await getDocs(q);
-    const clubs = snapshot.docs.map((club) => club.data() as Club);
-    return clubs;
+    const clubs = await this.getAllClubs();
```

Comment:
Is there potentially a way to query by name from the database, rather than by fetching all clubs and filtering them locally?

Reply:
Unfortunatly you can't do that with firestore